src/Page/Definition.elm

```diff
@@ -201,6 +218,86 @@ introStyle =
     Css.batch
         [ color colours.purple
         , fontFamilies [ "Raleway", sansSerif.value ]
-        , fontSize (rem 1.25)
+        , fontSize (rem 1)
         , margin2 (rem 2) zero
         ]
+
+
+expanderButtonStyle : Style
+expanderButtonStyle =
+    batch
+        [ alignItems center
+        , backgroundColor (hex "4f2f8d")
+        , border zero
+        , justifyContent spaceBetween
+        , textAlign left
+        , displayFlex
+        , padding (rem 0.5)
+        , width (pct 100)
+        ]
+
+
+expanderHeadingStyle : Style
+expanderHeadingStyle =
+    batch
+        [ color colours.white
+        , fontSize (rem 1.25)
+        , flex2 (int 1) (int 1)
+        ]
+
+
+expanderSymbolStyle : Style
+expanderSymbolStyle =
+    batch
+        [ color colours.white
+        , flex3 zero (int 1) (rem 3)
+        , textAlign center
+        , fontWeight (int 700)
+        , fontSize (rem 2.5)
+        , lineHeight (int 1)
+        , transform (rotate (deg 0))
+        , transition
+            [ Css.Transitions.transform 200
+            ]
+        ]
+
+
+rotate90Style : Style
+rotate90Style =
+    batch
+        [ transform (rotate (deg 90))
+        , transition
+            [ Css.Transitions.transform 200
+            ]
+        ]
+
+
+expanderItemStyle : Style
+expanderItemStyle =
+    batch [ marginTop (rem 1) ]
+
+
+expanderDefinitionStyle : List Style
+expanderDefinitionStyle =
+    [ batch
+        [ backgroundColor colours.lightgrey
+        , borderBottomLeftRadius (rem 1)
+        , borderBottomRightRadius (rem 1)
+        , padding (rem 1)
+        ]
+
+    -- Allow more padding space on larger screens
+    , withMedia [ only screen [ Media.minWidth (px 576) ] ]
+        [ padding (rem 2) ]
+    ]
+
+
+quoteStyle : List Style
+quoteStyle =
+    [ batch
+        [ fontSize (rem 1.1)
+        , fontWeight (int 300)
+        ]
+    , before [ property "content" "'\"'" ]
```

Comment:
That's a lot of escaping and " . Ha ha. Does `&quot;` or UTF char work? I mean really I don't mind.

Reply:
When I use the unicode `\u{0022}` escape, it's auto translated to `"` by my editor..

Reply:
Nice.